### PR TITLE
updated docs according to recent chages

### PR DESCRIPTION
**Pull request by masci**
### What does this PR do?

Docs updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,10 +13,15 @@ the latest version of the Agent.
 **Note:** the source code of the **stable** Datadog Agent 5 is located in the
 [dd-agent](https://github.com/DataDog/dd-agent) repository.
 
+## Documentation
```
Comment (masci) [inline README.md]: Moved this on top for better visibility

File: docs/beta.md [hunk 2]
```diff
@@ -129,25 +128,6 @@ find it with `sudo find / -type f -name 'tools.jar'`.
...
-Beta is currently available on these platforms:
-
-* Debian x86_64 version 7 (wheezy) and above (we do not support SysVinit)
-* Ubuntu x86_64 version 12.04 and above
-* RedHat/CentOS x86_64 version 6 and above
-* SUSE Enterprise Linux x86_64 version 11 SP4 and above (we do not support SysVinit)
-* MacOS 10.10 and above
-* Windows Server 64-bit 2008 R2 and above
-
-## Dogstatsd unix socket rights
```
Comment (masci) [inline docs/beta.md]: fixed

File: docs/beta.md [hunk 3]
```diff
@@ -129,25 +128,6 @@ find it with `sudo find / -type f -name 'tools.jar'`.
 Note: you may wish to specify alternative JVM heap parameters `-Xmx`, `-Xms`, the
 values used in the example correspond to the JMXFetch defaults.
 
-## Systems
```
Comment (masci) [inline docs/beta.md]: preparing to kill this page, moving away contents that'll be useful after the GA release too

File: docs/agent/changes.md [hunk 4]
```diff
@@ -99,7 +126,7 @@ The agent has a new set of command-line options:
 | version         | Print the version info |
 
 #### Gui
-* The browser session must be restarted each time the Agent service is restarted.  This will be fixed in an upcoming beta.
+* The browser session must be restarted each time the Agent service is restarted. This will be fixed in an upcoming release.
```
Comment (olivielpeau) [inline docs/agent/changes.md]: is this still true @derekwbrown?

**Review by olivielpeau — DISMISSED** [review]
💯

File: docs/agent/changes.md (hunk 4 above)
Comment (ofek) [reply docs/agent/changes.md]: This is fixed

https://github.com/DataDog/datadog-agent/pull/1180
https://github.com/DataDog/datadog-agent/pull/915

File: docs/agent/changes.md [hunk 5]
```diff
@@ -98,27 +95,80 @@ The agent has a new set of command-line options:
...
+
+### Linux
+
+There are a few major changes:
+
+* only the _lifecycle commands_ (i.e. `start`/`stop`/`restart`/`status` on the Agent service) should be run with `sudo service`/`sudo initctl`/`sudo systemctl`
+* all the other commands need to be run with the `datadog-agent` command, located in the `PATH` (`/usr/bin`) by default. The binary `dd-agent` is not available anymore.
+* the `info` command has been renamed `status`
+* the Agent 6 does not ship a SysV-init script (previously located at `/etc/init.d/datadog-agent`)
+* The configuration GUI is disabled by default on Linux-based systems.
```
Comment (olivielpeau) [inline docs/agent/changes.md]: the GUI on Linux isn't really a change (there never was a GUI on Linux for A5). Could we move this note to the `GUI` section?

File: docs/agent/changes.md [hunk 6]
```diff
@@ -98,27 +95,80 @@ The agent has a new set of command-line options:
...
+* The configuration GUI is now a web-based configuration application, point a
+web browser to `http://localhost:5002` to access it.
 
 ### MacOS
 
 * the _lifecycle commands_ (former `datadog-agent start`/`stop`/`restart`/`status` on the Agent 5) are replaced by `launchctl` commands on the `com.datadoghq.agent` service, and should be run under the logged-in user. For these commands, you can also use the Datadog Agent systray app
 * all the other commands can still be run with the `datadog-agent` command (located in the `PATH` (`/usr/local/bin/`) by default)
 * the `info` command has been renamed `status`
+* The configuration GUI is now a web-based configuration application, point a
+web browser to `http://localhost:5002` to access it.
```
Comment (olivielpeau) [inline docs/agent/changes.md]: we could also mention running `datadog-agent launch-gui` directly from a shell and using `Open Web UI` on the systray app. Easier to remember than the port IMO :)

**Review by olivielpeau — DISMISSED** [review]
A couple of comments, feel free to ship as soon as they're addressed

**Pull request merged**